File: bench/src/go/fib.go
package main
import "os"
func fib(n uint64) uint64 { if n<2 {return n}; return fib(n-1)+fib(n-2) }
func main(){ var s uint64; for i:=uint64(0);i<32;i++ { s+=fib(i) }; os.Exit(int(s%256)) }

File: bench/src/rust/fib.rs
fn fib(n:u64)->u64{ if n<2 {n} else {fib(n-1)+fib(n-2)} }
fn main(){ let mut s:u64=0; for i in 0..32u64 { s+=fib(i); } std::process::exit((s%256) as i32); }
